src/views/CheckoutOverview.vue

```diff
@@ -93,7 +99,7 @@ export default class CheckoutOverview extends Vue {
 
     @Watch('height')
     private logHeightChange(height: number, oldHeight: number) {
-        console.log(`Height changed from ${oldHeight} to ${height}`);
+        console.debug(`Got height: ${height} (was ${oldHeight})`);
```

Comment:
is this needed? aside from helping us developing it, this watcher probably should be removed prior to release.